redbot/cogs/trivia/data/lists/minecraft.yml

```diff
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
```

Comment:
Wouldn't it make sense to also add his actual name too? because thats just his nickname but alot knows him by his actual name over his nickname.

Reply:
Actually yes, he's mostly known as Markus "Notch" Persson. But yes I'd suggest to add in his real name instead of just the nickname i.e., Markus Alexej Persson.